radio/src/opentx.cpp

```diff
@@ -1070,7 +1070,20 @@ void getADC()
     if (x < 4)
       v = get_flysky_hall_adc_value(x) >> (1 - ANALOG_SCALE);
     else
-      v = getAnalogValue(x) >> (1 - ANALOG_SCALE);
+      #if defined(RADIO_TX16S)
+        switch (x)
+        {
+            case 9:
+            case 10:
```

Comment:
Is it possible to use named constants instead? This smells like a *magic value* 🤓

Reply:
Yeah, I did not know better how to solve this. Do you have tips?

Reply:
Maybe I got it - use EXT3 & EXT4 from enum Analogs